Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/ORTHelpers.cpp

```diff
@@ -0,0 +1,159 @@
+#include "pch.h"
+#include "ORTHelpers.h"
+#undef min
```

Comment:
If you declare `#define NOMINMAX` in your precompiled header file before `#include`ing `Windows.h`, the explicit undef should be unnecessary.

Reply:
Done.